pkgs/applications/graphics/rapid-photo-downloader/default.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv, fetchurl, python3Packages
+, file, intltool, gobjectIntrospection, libgudev
+, udisks, glib, gnome3, gst_all_1, libnotify
+, exiv2, exiftool, qt5, gdk_pixbuf
+}:
+
+python3Packages.buildPythonApplication rec {
+  name    = "rapid-photo-downloader-${version}";
+  version = "0.9.9";
+
+  src = fetchurl {
+    url = "https://launchpad.net/rapid/pyqt/${version}/+download/${name}.tar.gz";
+    sha256 = "193l55wdxwxmammyxwzp90sg0qw5r316443la3m9611sdpcmy8w3";
+  };
+
+  # Disable version check and fix install tests
+  postPatch = ''
+    substituteInPlace raphodo/constants.py \
+      --replace "disable_version_check = False" "disable_version_check = True"
+    substituteInPlace raphodo/rescan.py \
+      --replace "from preferences" "from raphodo.preferences"
+    substituteInPlace raphodo/copyfiles.py \
+      --replace "import problemnotification" "import raphodo.problemnotification"
+  '';
+
+  nativeBuildInputs = [ file intltool ];
+
+  buildInputs = [
+    gobjectIntrospection
+    (libgudev.overrideAttrs (oldAttrs: rec {
```

Comment:
Can you add a comment explaining why this is needed?

Reply:
Sure, no problem.